concepts/50 React Components/05 Add DevExtreme to a React Application/40 Import Stylesheets.md

```diff
@@ -5,6 +5,18 @@ Open the main component file (`App.js`) and import `dx.common.css` and a [predef
     import 'devextreme/dist/css/dx.common.css';
     import 'devextreme/dist/css/dx.light.css';
 
+Then, go to the `public` folder, open the `index.html` file, and add the `dx-viewport` class to the `<body>` tag. This ensures that theme colors and typography settings are applied to all elements on the page, not only to DevExtreme widgets.
+
+    <!-- tab: index.html -->
+    <html lang="en">
+        <head>
+            <!-- ... -->
+        </head>
+        <body class="dx-viewport">
+            <div id="root"></div>
+        </body>
+    </html>
+
 [note] [SVG-based widgets](/concepts/60%20Themes%20and%20Styles/00%20HTML-%20and%20SVG-Based%20Widgets.md '/Documentation/Guide/Themes_and_Styles/HTML-_and_SVG-Based_Widgets/') do not require theme stylesheets. If you do import the stylesheets, the widgets apply an appearance that matches them.
```

Comment:
Can you clarify what you're trying to say here (for my understanding)?
So, in the first sentence you say that SVG-based widgets don't require stylesheets, but in the second you mention that you can import stylesheets?

Reply:
SVG-based widgets work without stylesheets, but you may import the stylesheets for HTML-based widgets, in which case SVG-based widgets still do not use the stylesheets, but apply a similar appearance using their own theming mechanism.

Reply:
Can we say:
[SVG-based widgets](/concepts/60%20Themes%20and%20Styles/00%20HTML-%20and%20SVG-Based%20Widgets.md '/Documentation/Guide/Themes_and_Styles/HTML-_and_SVG-Based_Widgets/') do not require theme stylesheets. If you choose to import stylesheets, widgets apply an appearance that matches them.